setup.py

```diff
@@ -238,8 +238,12 @@ def setup_package():
         FULLVERSION, GIT_REVISION = get_version_info()
         metadata['version'] = FULLVERSION
     else:
-        if len(sys.argv) >= 2 and sys.argv[1] == 'bdist_wheel':
-            # bdist_wheel needs setuptools
+        if (len(sys.argv) >= 2 and sys.argv[1] == 'bdist_wheel' or
+                sys.version_info[0] < 3 and sys.platform == "win32"):
```

Comment:
because that forces the sdk use, it just sets a bunch of variables and assumes its there, it doesn't check the registry etc.
I'm guessing setuptools is smarter about it but I didn't check
for python3 I don't know I actually have vs2010 and so far I know there is no 2010 python sdk.
didn't try python3.5